### ISSUE TITLE
Accessibility issues for Create Namespace

### ISSUE THREAD

**Issue by nirajCITZ** [bcgov/api-services-portal#796]
1)
Description:  Ensures the contrast between foreground and background colors meets WCAG 2 AAA enhanced contrast ratio thresholds
Helpurl:      https://dequeuniversity.com/rules/axe/4.7/color-contrast-enhanced?application=axeAPI
[
    {
        "any": [
            {
                "id": "color-contrast-enhanced",
                "data": {
                    "fgColor": "#a9bacb",
                    "bgColor": "#003060",
                    "contrastRatio": 6.64,
                    "fontSize": "12.0pt (16px)",
                    "fontWeight": "normal",
                    "messageKey": null,
                    "expectedContrastRatio": "7:1"
                },
                "relatedNodes": [
                    {
                        "html": "<button type=\"button\" class=\"chakra-button css-qfqqp2\" data-testid=\"ns-modal-create-btn\">Create Namespace</button>",
                        "target": [
                            ".css-qfqqp2"
                        ]
                    }
                ],
                "impact": "serious",
                "message": "Element has insufficient color contrast of 6.64 (foreground color: #a9bacb, background color: #003060, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 7:1"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<button type=\"button\" class=\"chakra-button css-qfqqp2\" data-testid=\"ns-modal-create-btn\">Create Namespace</button>",
        "target": [
            ".css-qfqqp2"
        ],
        "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 6.64 (foreground color: #a9bacb, background color: #003060, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 7:1"
    }
]
![image.png](https://images.zenhubusercontent.com/61a121c4a7a3264dfdc530ef/212b9a29-55ff-4393-ad82-3a4f3ea87836)
------------------------------------------------------------
Description:  Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds
Helpurl:      https://dequeuniversity.com/rules/axe/4.7/color-contrast?application=axeAPI
[
    {
        "any": [
            {
                "id": "color-contrast",
                "data": {
                    "fgColor": "#505865",
                    "bgColor": "#b9c2d0",
                    "contrastRatio": 3.99,
                    "fontSize": "15.0pt (20px)",
                    "fontWeight": "normal",
                    "messageKey": null,
                    "expectedContrastRatio": "4.5:1"
                },
                "relatedNodes": [
                    {
                        "html": "<section class=\"chakra-modal__content css-i2828\" role=\"dialog\" id=\"chakra-modal-7\" tabindex=\"-1\" aria-modal=\"true\" style=\"opacity: 0.664619; transform: scale(0.983231) translateZ(0px);\" aria-labelledby=\"chakra-modal--header-7\" aria-describedby=\"chakra-modal--body-7\">",
                        "target": [
                            "#chakra-modal-7"
                        ]
                    },
                    {
                        "html": "<div class=\"chakra-modal__overlay css-hdd9l7\" style=\"opacity: 0.36151;\"></div>",
                        "target": [
                            ".chakra-modal__overlay"
                        ]
                    },
                    {
                        "html": "<nav role=\"banner\" class=\"css-f4yi47\">",
                        "target": [
                            "nav"
                        ]
                    }
                ],
                "impact": "serious",
                "message": "Element has insufficient color contrast of 3.99 (foreground color: #505865, background color: #b9c2d0, font size: 15.0pt (20px), font weight: normal). Expected contrast ratio of 4.5:1"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<header class=\"chakra-modal__header css-p9al81\" id=\"chakra-modal--header-7\">Create Namespace</header>",
        "target": [
            "#chakra-modal--header-7"
        ],
        "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.99 (foreground color: #505865, background color: #b9c2d0, font size: 15.0pt (20px), font weight: normal). Expected contrast ratio of 4.5:1"
    },
    {
        "any": [
            {
                "id": "color-contrast",
                "data": {
                    "fgColor": "#4a5461",
                    "bgColor": "#a9b8c6",
                    "contrastRatio": 3.79,
                    "fontSize": "12.0pt (16px)",
                    "fontWeight": "normal",
                    "messageKey": null,
                    "expectedContrastRatio": "4.5:1"
                },
                "relatedNodes": [
                    {
                        "html": "<input name=\"name\" type=\"text\" data-testid=\"ns-modal-name-input\" id=\"field-103\" required=\"\" aria-required=\"true\" class=\"chakra-input css-345m2s\" aria-describedby=\"field-103-helptext\">",
                        "target": [
                            "#field-103"
                        ]
                    }
                ],
                "impact": "serious",
                "message": "Element has insufficient color contrast of 3.79 (foreground color: #4a5461, background color: #a9b8c6, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<input name=\"name\" type=\"text\" data-testid=\"ns-modal-name-input\" id=\"field-103\" required=\"\" aria-required=\"true\" class=\"chakra-input css-345m2s\" aria-describedby=\"field-103-helptext\">",
        "target": [
            "#field-103"
        ],
        "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.79 (foreground color: #4a5461, background color: #a9b8c6, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
    },
    {
        "any": [
            {
                "id": "color-contrast",
                "data": {
                    "fgColor": "#396088",
                    "bgColor": "#a9b8c6",
                    "contrastRatio": 3.23,
                    "fontSize": "12.0pt (16px)",
                    "fontWeight": "normal",
                    "messageKey": null,
                    "expectedContrastRatio": "4.5:1"
                },
                "relatedNodes": [
                    {
                        "html": "<button type=\"button\" class=\"chakra-button css-1ynby2t\" data-testid=\"ns-modal-cancel-btn\">Cancel</button>",
                        "target": [
                            ".css-1ynby2t"
                        ]
                    }
                ],
                "impact": "serious",
                "message": "Element has insufficient color contrast of 3.23 (foreground color: #396088, background color: #a9b8c6, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<button type=\"button\" class=\"chakra-button css-1ynby2t\" data-testid=\"ns-modal-cancel-btn\">Cancel</button>",
        "target": [
            ".css-1ynby2t"
        ],
        "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.23 (foreground color: #396088, background color: #a9b8c6, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
    }
]
![image.png](https://images.zenhubusercontent.com/61a121c4a7a3264dfdc530ef/bd4b7ea9-212a-4928-852e-cd19d18dd82a)
